marc.go

```diff
@@ -423,6 +420,11 @@ func getHoldings(fmlRecord fml.Record, tag string, subfieldCodes []string) []Hol
 			CallNumber: subfieldValue(f.SubFields, subfieldCodes[2]),
 			Summary:    subfieldValue(f.SubFields, subfieldCodes[3]),
 			Notes:      subfieldValue(f.SubFields, subfieldCodes[4])}
+		if tag == "866" {
+			holding.Format = "Print volume"
```

Comment:
This makes me nervous. We've been told with certainty they never include 866 for electronic serials?

Reply:
Yes, I was told it is only used for print materials. I confirmed this with a quick test of our latest full dump as well.